SingularityUI/app/templates/taskDetail/taskHealthChecks.hbs

```diff
@@ -0,0 +1,55 @@
+{{#if synced}}
+    {{#ifCoercedBooleanIsTrue data.task.taskRequest.deploy.healthcheckUri}}
```

Comment:
i think we can get away with just `{{#if data.task.taskRequest.deploy.healthcheckUri}}` here -- is there a situation where it doesnt?